rskj-core/src/test/java/co/rsk/peg/lockingcap/LockingCapSupportImplTest.java

```diff
@@ -0,0 +1,302 @@
+
+        // Assert
+        assertFalse(actualResult);
+        assertEquals(Optional.of(constants.getInitialValue()), lockingCapSupport.getLockingCap());
+    }
+
+    @Test
+    void increaseLockingCap_whenAPreviousValueExistsInStorageAndNewLockingCapIsGreaterThanPreviousValue_shouldReturnTrue() {
+        // Arrange
+        Coin previousLockingCap = constants.getInitialValue().add(Coin.SATOSHI);
+        lockingCapStorageProvider.setLockingCap(previousLockingCap);
+        lockingCapSupport.save();
+        // Recreate LockingCapSupport to load the previous value from storage
+        lockingCapStorageProvider = new LockingCapStorageProviderImpl(bridgeStorageAccessor);
+        lockingCapSupport = new LockingCapSupportImpl(lockingCapStorageProvider, activations, constants, signatureCache);
+
+        Coin newLockingCap = previousLockingCap.add(Coin.SATOSHI);
+        Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, LockingCapCaller.AUTHORIZED.getRskAddress());
+
+        // Act
+        boolean actualResult = lockingCapSupport.increaseLockingCap(tx, newLockingCap);
+
+        // Assert
+        assertTrue(actualResult);
+        assertEquals(Optional.of(newLockingCap), lockingCapSupport.getLockingCap());
+    }
+
+    @Test
+    void increaseLockingCap_whenAPreviousValueExistsInStorageAndNewLockingCapIsLessThanPreviousValue_shouldReturnFalse() {
+        // Arrange
+        Coin previousLockingCap = constants.getInitialValue().add(Coin.SATOSHI);
+        lockingCapStorageProvider.setLockingCap(previousLockingCap);
+        lockingCapSupport.save();
+        // Recreate LockingCapSupport to load the previous value from storage
+        lockingCapStorageProvider = new LockingCapStorageProviderImpl(bridgeStorageAccessor);
+        lockingCapSupport = new LockingCapSupportImpl(lockingCapStorageProvider, activations, constants, signatureCache);
+
+        Coin newLockingCap = previousLockingCap.subtract(Coin.SATOSHI);
+        Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, LockingCapCaller.AUTHORIZED.getRskAddress());
+
+        // Act
+        boolean actualResult = lockingCapSupport.increaseLockingCap(tx, newLockingCap);
+
+        // Assert
+        assertFalse(actualResult);
+        assertEquals(Optional.of(previousLockingCap), lockingCapSupport.getLockingCap());
+    }
+
+    @Test
+    void increaseLockingCap_whenAnUnauthorizedCallerRequestToIncreaseLockingCapValue_shouldReturnFalse() {
+        // Arrange
+        Coin newLockingCap = constants.getInitialValue().add(Coin.SATOSHI);
+        Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, LockingCapCaller.UNAUTHORIZED.getRskAddress());
+
+        // Act
+        boolean actualResult = lockingCapSupport.increaseLockingCap(tx, newLockingCap);
+
+        // Assert
+        assertFalse(actualResult);
+        assertEquals(Optional.of(constants.getInitialValue()), lockingCapSupport.getLockingCap());
```

Comment:
if you are asserting these both things, i think the test name should say `_shouldDoNothing` (or something like that, something more general)

Reply:
No agree. The main `assertion` is related to `assertFalse` since it is the `actualResult` of the method we are testing. The another `assertion` is important because it adds value.

Reply:
`shouldReturnFalse` or `shouldReturnFalseAndDoNothing` both seem ok to me. I wouldn't complicate things too much with that.

Reply:
yep, agree. No worries